ale_linters/swift/swiftlint.vim

```diff
@@ -1,6 +1,24 @@
-" Author: David Mohundro <david@mohundro.com>
+" Author: David Mohundro <david@mohundro.com>, Gordon Fontenot <gordon@fonten.io>
 " Description: swiftlint for swift files
 
+call ale#Set('swift_swiftlint_executable', 'swiftlint')
+call ale#Set('swift_swiftlint_use_global', get(g:, 'ale_use_global_executables', 0))
+
+function! ale_linters#swift#swiftlint#GetExecutable(buffer) abort
+  return ale#node#FindExecutable(a:buffer, 'swift_swiftlint', [
```

Comment:
Is it worth pulling this out of `ale#node` and into `ale#utils`? It's being used in a bunch of different linters that have nothing to do with node, and there doesn't _seem_ to be anything about it that's specific to node itself. There's also `ale#python#FindExecutable`, which might be able to be removed and replaced with this common util function.

Reply:
I believe both of those functions used to be specific to virtual environments and node_modules before being generalized. I'd be happy to take a follow-up PR factoring them out into a more generic, single function.